client.go

```diff
@@ -46,7 +63,7 @@ func NewClient(authToken string, accountGroupID string) *Client {
 		AccountGroupID: accountGroupID,
 		APIEndpoint:    apiEndpoint,
 		HTTPClient: http.Client{
-			Timeout: time.Second * 10,
+			Timeout: time.Second * 20,
```

Comment:
Should we make this configurable somehow ?  Maybe overridden via env arg?

Reply:
Absolutely agree.  But since there may be different opinions on how, I figured it belonged in a separate PR.

I should have caught this before submitting.  Though based on observed response times, this should probably be the minimum.